cinema/urls.py

```diff
@@ -1,10 +1,43 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    ActorList,
+    ActorDetail,
+    GenreDetail,
+    GenreList,
+    CinemaHallViewSet
+)
+from .views import MovieViewSet
 
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(router.urls)),
+    path("actors/", ActorList.as_view(), name="actor-list"),
+    path("actors/<int:pk>/", ActorDetail.as_view(), name="actor-detail"),
+    path("genres/", GenreList.as_view(), name="actor-list"),
+    path("genres/<int:pk>/", GenreDetail.as_view(), name="actor-detail"),
+    path(
```

Comment:
You can register CinemaHallViewSet in router with the correct prefix. It will look better